nnpackage/schema/circle_schema.fbs

```diff
@@ -1100,8 +1123,31 @@ table AssignVariableOptions {
 }
 
 table RandomOptions {
-  seed: int;
-  seed2: int;
+  seed: long;
+  seed2: long;
```

Comment:
`int` to `long` may break compatibility. I wondered it is okay. I searched history.

https://github.com/tensorflow/tensorflow/commit/6fdfac0f93b00c9e9feb151dac6a12dbf81732b0 says

>  Updating RandomOptions seed/seed2 types in the TFLite schema from int…
>
> … to long type.
>
> RandomOptions is not currently used by any ops, since converter changes didn't land  yet.